src/model/Assertion.lua

```diff
@@ -17,6 +17,8 @@
     * For example: r = sub, obj, act
     policy = {{}}
 ]]
+local Util = require("src/util/Util")
+
 local Assertion = {}
```

Comment:
The position of the comment of Assertion should not be changed. It should be like this:
```lua
local Util = require("src/util/Util")

--[[
    * Assertion represents an expression in a section of the model.
    * For example: r = sub, obj, act
    policy = {{}}
]]
local Assertion = {}

```

Reply:
Changed